### ISSUE TITLE
[Bug] Fast Leaves are broken

### ISSUE THREAD

**Issue by guivnf** [Potion-Studios/Oh-The-Biomes-Weve-Gone#132]
Basically when enabling fast leaves, instead of making their transparent parts opaque/darken, they continue to be transparent, but without rendering the other side.

With Fast graphics:
![image](https://github.com/user-attachments/assets/07b0b968-0155-4303-b0c7-8db88703eec8)

Without
![image](https://github.com/user-attachments/assets/6ea81c60-81f0-4f06-8823-98399143f4fb)

Expected behaviour: 
![image](https://github.com/user-attachments/assets/9e0d694d-fa80-4a08-99a1-a9c5379134b9)

**Comment by JT122406** [comment]
I will have to dive into this since we just set the render type with Mc normally it handles the rest so idk if it’s a midlander issue or something, what platform and Mc version are you using/have you tried any other platform or version

**Comment by guivnf** [comment]
> I will have to dive into this since we just set the render type with Mc normally it handles the rest so idk if it’s a midlander issue or something, what platform and Mc version are you using/have you tried any other platform or version

I've tried using the default forge launcher, on 47.2.20, 1.20.1. Switched launchers and also disabled all mods aside OhTheBiomes...

I've actually found out a bit about why this issue happens, apparently bwg doesn't define the rendering method in the block model for it's leaves. Take a look at how acacia leaves are structured:
```json
{
  "parent": "minecraft:block/leaves",
  "textures": {
    "all": "minecraft:block/acacia_leaves"
  }
}
```
After i added that in a resourcepack it kinda works, but the thing is, the inbetweens (where it's currently transparent), become all black and not exactly a dark greenish as other leaves, so yeah, that's all i could gather out of it

**Comment by JT122406** [comment]
We do the samething as normal mc
{
  "parent": "minecraft:block/leaves",
  "render_type": "minecraft:cutout_mipped",
  "textures": {
    "all": "biomeswevegone:block/fir/leaves"
  }
}
We also add in the render type in json as that's the way forge wants you to do rendertypes

**Comment by guivnf** [comment]
> We do the samething as normal mc
> 
> {
> 
>   "parent": "minecraft:block/leaves",
> 
>   "render_type": "minecraft:cutout_mipped",
> 
>   "textures": {
> 
>     "all": "biomeswevegone:block/fir/leaves"
> 
>   }
> 
> }
> 
> We also add in the render type in json as that's the way forge wants you to do rendertypes

That's odd, hmm are you able to replicate this issue in your side ? Maybe it's something on my end

**Comment by JT122406** [comment]
So I can recreate it on NeoForge and Forge but on fabric it works fine which leaves me to believe it's an issue with how Forge like platforms handle the rendertype changing during graphic settings changes

**Comment by JT122406** [comment]
I'm gonna close this as there's nothing we can do on our side unless we use the Deprecated by forge way of using the vanilla map.  This should be reported to forge as a issue with the rendertype not changing with the graphics properties

**Comment by guivnf** [comment]
> I'm gonna close this as there's nothing we can do on our side unless we use the Deprecated by forge way of using the vanilla map. This should be reported to forge as a issue with the rendertype not changing with the graphics properties

anything i can do? Should i update forge or something like that?

**Comment by JT122406** [comment]
No It just needs to be fixed by the forge team

**Comment by Onigeshi** [comment]
Hi, first of all sorry for "reviving" this, but I've encountered this recently as well and started looking to see if I can fix it with a resourcepack.. and it looks like removing the `"render_type": "minecraft:cutout_mipped",` from the models fixes it, at least with NeoForge on 1.21.1 (ver. 21.1.115). I haven't tested with fabric or other versions.

I'm not a modder but reading the documentation it looks like this field is optional and so far flying around the world I don't see any rendering weirdness of the leaves after removing this field.

https://docs.neoforged.net/docs/resources/client/models/#render-types

fast (tested using aspen leaf):
![Image](https://github.com/user-attachments/assets/cfc44519-79b7-4dab-ab86-ff188f0f39af)

fancy (tested using aspen leaf):
![Image](https://github.com/user-attachments/assets/dfc2674a-f17e-433e-9b88-0fa5cff9e2c7)

fast settings flying over an entire zelkova forest after removing the render_type field:
![Image](https://github.com/user-attachments/assets/7221884f-533d-4b12-8260-d94393cc71d1)


Edit: after more testing, the only leaves having issues by removing the render_type are the Yucca Leaves 

![Image](https://github.com/user-attachments/assets/e045f278-0bad-484d-946e-4e9718fe6cbe)

**Comment by JT122406** [comment]
NeoForge has changed somethings so I will look into it on there side, Forge I am working with on Fixing that can be found here and will be fixed once this pr is merged
1.21.1 https://github.com/MinecraftForge/MinecraftForge/pull/10396
1.20.1 https://github.com/MinecraftForge/MinecraftForge/pull/10393

**Comment by JT122406** [comment]
Fixes have been Merged into Forge Starting with Version 47.3.32 (1.20.1) and 52.0.49 (1.21.1)
